marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/IngestionStepDefinitionImpl.java

```diff
@@ -30,11 +32,21 @@
     public IngestionStepDefinitionImpl(String name) {
         setName(name);
         setType(StepDefinitionType.INGESTION);
+
+        JSONObject jsonObject = new JSONObject();
+        jsonObject.put("inputFilePath", "");
+        jsonObject.put("outputURIReplacement", "");
+        jsonObject.put("inputFileType", "");
+        fileLocations = jsonObject.jsonNode();
         setFileLocations(fileLocations);
 
```

Comment:
https://github.com/marklogic/marklogic-data-hub/blob/805acf5f716ec7e46079abfadb150d2bd5348d69/marklogic-data-hub/src/main/java/com/marklogic/hub/step/impl/IngestionStepDefinitionImpl.java#L40-L41

this.fileLocations = jsonObject.jsonNode();